developing_collections_structure.rst: Add EE collection-level metadata link to meta directory subsection (#855)
ansible/ansible-documentation: docs/docsite/rst/dev_guide/developing_collections_structure.rst

         - my_module
         - another.collection.another_module

execution_environments.yml
^^^^^^^^^^^^^^^^^^^^^^^^^^

> If your collection has requirements, you can specify them in the ``execution-environment.yml`` file  in the ``meta`` directory not to make users do it manually when building `Execution Environments <https://docs.ansible.com/ansible/devel/getting_started_ee/index.html>`_ containing your collection.

Reviewer: ```suggestion
If your collection has requirements, you can specify them in the ``execution-environment.yml`` file  in the ``meta`` directory. This ensures users do not need to add these requirements manually when building `Execution Environments <https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html>`_ containing your collection.
```

> If your collection has requirements, you can specify them in the ``execution-environment.yml`` file  in the ``meta`` directory. This ensures users do not need to add these requirements manually when building `Execution Environments <https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html>`_ containing your collection.

Author: ```suggestion
If your collection has requirements, you can specify them in the ``execution-environment.yml`` file  in the ``meta`` directory.
This ensures users do not need to add these requirements manually when building `Execution Environments <https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html>`_ containing your collection.
```
